src/main/kotlin/org/taktik/icure/services/external/rest/v2/dto/embed/TimeTableItemDto.kt

```diff
@@ -26,11 +26,12 @@ import com.github.pozo.KotlinBuilder
 @JsonIgnoreProperties(ignoreUnknown = true)
 @KotlinBuilder
 data class TimeTableItemDto(
-	val days: List<String> = emptyList(),
+	val rrule: String? =null,
```

Comment:
Could you please add some KDoc to explain what it the rrule purpose and which format it needs to follow to replace days & recurrenceTypes properly ?
Precise it follows the standard RFC-5545